Cargo.toml

```diff
@@ -263,7 +263,7 @@ feat_os_windows_legacy = [
 test = [ "uu_test" ]
 
 [workspace.dependencies]
-atty = "0.2"
```

Comment:
Nit: This list was ordered alphabetically :)

Reply:
I will fix this.